examples/polyline_text_path_example.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+
+# This is to import the repository's version of folium ; not the installed one.
+import sys, os
+sys.path.insert(0, '..')
```

Comment:
I think we can drop these two lines. (BTW I am planning to re-do all examples as notebooks that we can run on Travis as "visual"  tests.)